gmail-to-salesforce-lead/main.bal

```diff
@@ -0,0 +1,193 @@
+//Re-order the imports alphabetically
+import ballerina/lang.runtime;
+import ballerina/log;
+import ballerina/mime;
+import ballerinax/googleapis.gmail as gmail;
```

Comment:
```suggestion
import ballerinax/googleapis.gmail;
```

Defaults to `gmail`.

Reply:
Fixed in [cce2eaf](https://github.com/ballerina-guides/integration-samples/pull/32/commits/cce2eafff5580b9fc4de69c431a8951e6c90b36a)